lib/membrane_mp4/muxer/cmaf/track_samples_queue.ex

```diff
@@ -326,4 +326,8 @@ defmodule Membrane.MP4.Muxer.CMAF.TrackSamplesQueue do
         excess_samples: []
     }
   end
+
+  defp key_frame?(%{metadata: %{h264: %{key_frame?: true}}}), do: true
+  defp key_frame?(%{metadata: %{h265: %{key_frame?: true}}}), do: true
+  defp key_frame?(_sample), do: false
```

Comment:
Seems like this kind of function is repeated many times, maybe let's add a `codec_helper.ex` and put it there? I guess it should accept just the metadata to be most flexible.

Reply:
I added the function to the already existing `Membrane.MP4.Helper` module.